Source/RageCommunity.Library/Extensions/VectorExtensions.cs

```diff
@@ -25,5 +32,24 @@ public static Vector3 HeightToGround(this Vector3 position)
 
             return position;
         }
+
+        /// <summary>
+        /// Gets an offset position from the current position by a given heading.
+        /// </summary>
+        /// <remarks>
+        /// Written by alexguirre. 
+        /// </remarks>
+        internal static Vector3 GetOffset(this Vector3 from, float heading, Vector3 offset)
```

Comment:
suggestion: public instead of internal
question: what exactly does the offset parameter?

Reply:
Oops, copy/paste mistake.  The offset parameter is the distance the offset position will be from the original position.